.gitignore

```diff
@@ -0,0 +1,4 @@
+.terraform.lock.hcl
+.terraform/
+terraform.tfstate
+terraform.tfstate.backup
```

Comment:
could you  please, add newline at the end of file

Reply:
Added a new line EOF.